ocp_resources/machine_config_pool.py

```diff
@@ -7,3 +7,30 @@ class MachineConfigPool(Resource):
     class Status(Resource.Status):
         UPDATED = "Updated"
         UPDATING = "Updating"
+
+    def __init__(
```

Comment:
Please check other resources, all should be with default None and you have missing args that should be here.

Reply:
@myakove Which args are missing? Can you please provide a resource which I can use as a reference to complete the missing args?
And BTW - why all args should be with default None? Doesn't it make sense that some of them must be set by the caller?

Reply:
The user have option to send a YAML file, therefor no args needed.

Reply:
Ack, thanks.

Reply:
https://github.com/RedHatQE/openshift-python-wrapper/blob/main/CONTRIBUTING.md

Reply:
Thanks Meni, now I added the extra missing args.

Reply:
Ack